codalab/worker/main.py

```diff
@@ -184,6 +184,12 @@ def parse_args():
         default=3,
         help='The number of times to retry downloading dependencies after a failure (defaults to 3).',
     )
+    parser.add_argument(
+        '--shared-memory-size-gb',
+        type=int,
+        default=1,
```

Comment:
Maybe it's better to make this argument similar to other arguments with byte inputs (such as `--max-memory`)? So people would input things like `1g`.